README.md

```diff
@@ -1,23 +1,31 @@
-# Dogecoin Core [DOGE, Ð]
-==========================
+<p align="center">
+<img src="http://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png"
+</p>
+<h1 align="center">Dogecoin Core [DOGE, Ð]</h1>
 
-![Dogecoin](https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png)
+[![Build Status](https://travis-ci.com/dogecoin/dogecoin.svg?branch=master)](https://travis-ci.com/dogecoin/dogecoin) [![tip for next commit](https://tip4commit.com/projects/702.svg)](https://tip4commit.com/github/dogecoin/dogecoin)
 
-[![Build Status](https://travis-ci.org/dogecoin/dogecoin.svg?branch=1.14-dev)](https://travis-ci.org/dogecoin/dogecoin) [![tip for next commit](https://tip4commit.com/projects/702.svg)](https://tip4commit.com/github/dogecoin/dogecoin)
+## What is Dogecoin? – Such coin 🤔  
 
-## What is Dogecoin? – Such coin
-Dogecoin is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
+Dogecoin (styled as DOGE or Ð) is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
+- **Website:** [dogecoin.com.](https://dogecoin.com)
+- **Get started with doge here:** [dogecoin.com/getting-started.](https://dogecoin.com/getting-started.)
 
-https://dogecoin.com/
+## Wallets – Wow much hodl 🐕‍🦺
+### **Official Dogecoin Wallet** <img src="http://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" width="25" height="25">
+- Supports Windows, Linux, MacOS and Android!
+- Get your paws on the [**official download!**](dogecoin.com/getting-started)
```

Comment:
Not sure what this adds...

Reply:
There felt like there was no information of wallets for doge... I can remove if you feel it doesn't fit the flow or adds anything.

Reply:
Removed wallet from the readme.